osidb/tests/test_snippet.py

```diff
@@ -10,7 +10,7 @@
 pytestmark = pytest.mark.unit
 
 
-def get_snippet(cve_id="CVE-2023-0001"):
+def get_nvd_snippet(cve_id="CVE-2023-0001"):
```

Comment:
I do not get why should we rename this and everything below. Sure we have a single Snippet type right now but this is supposed to test the general Snippet operations which should be type-independent.